### PR TITLE
My draft summary

### PR DESCRIPTION
**Pull request by specfazhou**
it is my draft summary

### CODE REVIEW

File: _summaries/goodfellow2013maxout_2.md [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+* What is the core idea? <br/>
+This paper comes up with a new function -- maxout function. Using Maxout function together with dropout techniques can classify images with lowest error rate on several well-known images dataset at that time.
```
Comment (jaywhang) [inline _summaries/goodfellow2013maxout_2.md:13]: I feel that this doesn't adequately describe the core idea.  The fact that it leads to better experimental performance is a consequence of the idea.  I think that the core idea of maxout is to use a _learned_ the activation function that has the following properties:
* It is strictly more expressive than rectifiers (as Fig. 1 shows)
* It produces representations (i.e. layer outputs) that are less sparse, thus alleviating the dead neuron problem.

File: _summaries/goodfellow2013maxout_2.md [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+* What is the core idea? <br/>
+This paper comes up with a new function -- maxout function. Using Maxout function together with dropout techniques can classify images with lowest error rate on several well-known images dataset at that time. 
+
+* How is it realized (technically)? <br/>
+the input is a **d**-dim vertical vector, **X=(x_1,x_2,...x_d)^{t}**) with outputs **m**-dim vertical vector **h = (h_1,h_2,...,h_m)^{t}** in the next layer. Choose an integer **k**, and what the maxout function does is equivalent to adding another layer in between with **k** x **m** units, which forms a matrix **Z = (z_{i,j}**) where **0<i<k+1, 0<j<m+1**. Then, the **j**-th column of **Z** is **W(: , :, j)X**, where **W** is a **k** x **d** x **m** tensor with elements are parameters, and **h_j** is the maximum element in the **j**-th column of **Z**.
```
Comment (jaywhang) [inline _summaries/goodfellow2013maxout_2.md:16]: The detailed technical description is nice, but it feels a bit too dense with minute details.  Perhaps we can include a higher-level description of what it does.  Something like: "Each output neuron of a maxout layer is computed as the maximum of a single fully-connected layer.  Thus, the maxout layer can be seen as a concatenation of multiple fully-connected layers with "max" aggregation function."

Also, isn't the last **h_j** supposed to be **h_i**?

File: _summaries/goodfellow2013maxout_2.md [hunk 3]
```diff
@@ -1,19 +1,31 @@
...
+
+* How well does the paper perform?<br/>
+On MNIST, CIFAR-10, CIFAR-100, SVHN these four datasets, the neural networks that use maxout function and dropout techique lower the error rate compare to previous works. Especially on CIFAR-100, it reduced error rate from 42.51% to 38.57%. Although in some cases, the way they preprocess datas, and the size of models they use are different, they did comparison experiments to show Maxout function is the reason for better performance. 
+
+* What interesting variants are explored?<br/>
+It seems this paper only focuses on the maxout function, and mainly discuss how well it performed together with dropout technique.
```
Comment (jaywhang) [inline _summaries/goodfellow2013maxout_2.md:23]: Since the paper spends a decent amount of space introducing the universal approximability of maxout networks, perhaps it can be mentioned in 1-2 sentences somewhere.

File: _summaries/goodfellow2013maxout_2.md [hunk 4]
```diff
@@ -1,19 +1,31 @@
...
+It seems this paper only focuses on the maxout function, and mainly discuss how well it performed together with dropout technique.  
 
 ## TL;DR
 * Three
 * Bullets
-* To highlight the core concepts
+* To highlight the core concepts<br/>
+1. The maxout function has parameters can be learned, that's very different from traditional activication function like softmax, and sigmoid. <br/>
+2. With two maxout layers, one can approximate any continuous function on closed and bounded subset of **R^d**, and one maxout layer can approximate linearly piecewise function<br/>
+3. Compare to max{a,0} function, maxout function seems make the computer easier to do backpropagation.<br/>
```
Comment (jaywhang) [inline _summaries/goodfellow2013maxout_2.md:30]: I'm not sure what this means.  In what sense is maxout "easier" to perform backpropagation on.  Perhaps you meant that, when combined with dropout, maxout is more amenable to optimization in comparison to rectifiers?

File: _summaries/goodfellow2013maxout_2.md [hunk 5]
```diff
@@ -1,19 +1,31 @@
...
+
+* How is it realized (technically)? <br/>
+the input is a **d**-dim vertical vector, **X=(x_1,x_2,...x_d)^{t}**) with outputs **m**-dim vertical vector **h = (h_1,h_2,...,h_m)^{t}** in the next layer. Choose an integer **k**, and what the maxout function does is equivalent to adding another layer in between with **k** x **m** units, which forms a matrix **Z = (z_{i,j}**) where **0<i<k+1, 0<j<m+1**. Then, the **j**-th column of **Z** is **W(: , :, j)X**, where **W** is a **k** x **d** x **m** tensor with elements are parameters, and **h_j** is the maximum element in the **j**-th column of **Z**.      
+
+* How well does the paper perform?<br/>
+On MNIST, CIFAR-10, CIFAR-100, SVHN these four datasets, the neural networks that use maxout function and dropout techique lower the error rate compare to previous works. Especially on CIFAR-100, it reduced error rate from 42.51% to 38.57%. Although in some cases, the way they preprocess datas, and the size of models they use are different, they did comparison experiments to show Maxout function is the reason for better performance. 
+
+* What interesting variants are explored?<br/>
```
Comment (jaywhang) [inline _summaries/goodfellow2013maxout_2.md:21]: I think this is where we could discuss some of their detailed studies on the effect of maxout on  model averaging and optimization performance.

File: _summaries/goodfellow2013maxout_2.md (hunk 1 above)
Comment (specfazhou) [reply _summaries/goodfellow2013maxout_2.md:13]: Thanks for your advice, I agree with you that the core idea is not about performance but about maxout function.

File: _summaries/goodfellow2013maxout_2.md (hunk 2 above)
Comment (specfazhou) [reply _summaries/goodfellow2013maxout_2.md:16]: I think the last h_j is correct. Thanks for your advice about high-level description though, since the summary requires us to write technical description about maxout function, maybe I just don't change this time.

File: _summaries/goodfellow2013maxout_2.md (hunk 3 above)
Comment (specfazhou) [reply _summaries/goodfellow2013maxout_2.md:23]: That's one interesting point to add, and I added

File: _summaries/goodfellow2013maxout_2.md (hunk 4 above)
Comment (specfazhou) [reply _summaries/goodfellow2013maxout_2.md:30]: I have changed the third point, thank you

File: _summaries/goodfellow2013maxout_2.md (hunk 5 above)
Comment (specfazhou) [reply _summaries/goodfellow2013maxout_2.md:21]: I have added more details about the paper's performance. Since the question only asks how the paper perform, so I guess it is fine not to add contents in section 7 and 8

**Comment by zhaoyue-zephyrus** [comment]
Closed due to a newer PR on #20
